testsuite/features/secondary/srv_restart.feature

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Restart the spacewalk-services via UI
```

Comment:
You need to include this feature in https://github.com/uyuni-project/uyuni/blob/master/testsuite/run_sets/secondary.yml#L61

Reply:
done